docs/source/changelog.rst

```diff
@@ -12,6 +12,7 @@ Changelog
         * Prevent DFS from creating Identity Features of globally ignored variables (:pr:`819`)
     * Changes
         * Remove python 2.7 support from serialize.py (:pr:`812`)
+        * Make smart_open, boto3, and s3fs optional dependencies (:pr:`827`)
```

Comment:
maybe add a breaking change saying that you now have to install these yourself to use certain functionality